admin/src/main/java/edu/unc/lib/dl/admin/controller/PerformanceMonitorController.java

```diff
@@ -0,0 +1,299 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.admin.controller;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+@Controller
+public class PerformanceMonitorController {
+	private static final Logger log = LoggerFactory
+			.getLogger(PerformanceMonitorController.class);
+	
+	private static final String NEW_LINE_SEPARATOR = "\n";
+	private static final Object [] FILE_HEADERS = {
+		"date",
+		"uuid",
+		"throughput_files",
+		"throughput_bytes",
+		"queued_duration",
+		"ingest_duration",
+		"finished",
+		"moves",
+		"finished_enhancements",
+		"failed_enhancements",
+		"failed_deposit",
+		"failed_deposit_job"
+	};
+	private CSVFormat csvFileFormat = CSVFormat.DEFAULT.withRecordSeparator(NEW_LINE_SEPARATOR);
+	
+	@Autowired
+	private JedisPool jedisPool;
+	
+	public JedisPool getJedisPool() {
+		return jedisPool;
+	}
+
+	public void setJedisPool(JedisPool jedisPool) {
+		this.jedisPool = jedisPool;
+	}
+
+	public Set<String> getDepositMetrics() {
+		try (Jedis jedis = getJedisPool().getResource()){
+			return jedis.keys("deposit-metrics:*");
+		}
+	}
+	
+	private Boolean buildFile(String path) {
+		File csvFile = new File(path);
+		
+		if(csvFile.exists() && !csvFile.isDirectory()) { 
+			Long currentTime = System.currentTimeMillis();
+			Long fileCreationTime = csvFile.lastModified();
+			
+			if ((currentTime - fileCreationTime ) > 3600000) {
```

Comment:
I'd suggest making this value a constant at the top of the file, and it can be helpful to set it to 60 * 60 * 1000 to make it a little clearer how long the period is